### ISSUE TITLE
How to get the high performance on slow-fast model?

### ISSUE THREAD

**Issue by cecilia930426** [open-mmlab/mmaction2#829]
I used the slowfast_r101_8x8x1_256e_kinetics400_rgb and slowfast_r50_8x8x1_256e_kinetics400_rgb configs to train the slow-fast model. But I can only get 52% and 61% on top1 acc. Could you please help me find the reason?

**Comment by irvingzhang0512** [comment]
Could you share more details, such as dataset, training command, config file, modified settings, number of gpus, traceback info...

**Comment by cecilia930426** [comment]
> Could you share more details, such as dataset, training command, config file, modified settings, number of gpus, traceback info...

I trained slowfast model on activitynet dataset. The config file is the same as you provided.  slowfast_r101_8x8x1_256e_kinetics400_rgb.py slowfast_r50_8x8x1_256e_kinetics400_rgb.py. I didn't modify other setting.

**Comment by innerlee** [comment]
Is the num of gpus the same?

**Comment by cecilia930426** [comment]
> Is the num of gpus the same?

Yes

**Comment by congee524** [comment]
we train the two models on 8x4 8x3 GPUs. So the gradient accumulation and batch normalization may behave differently...
